content/docs/command-reference/plots/templates.md

```diff
@@ -0,0 +1,145 @@
+## plot templates
+
+Dump plot templates.
+
+Sometimes users need to customize the way project data is visualized. This
+command allows dumping templates used by DVC, so that the users don't have to
+build their own templates from zero.
+
+## Synopsis
+
+```usage
+usage: dvc plots templates [-h] [-q | -v] [-o <path>] [{simple,linear,confusion,confusion_normalized,scatter,smooth}]
+
+positional arguments:
+ template
```

Comment:
I see this was [suggested into lower case](https://github.com/iterative/dvc.org/pull/3691#pullrequestreview-1020932156) but to be consistent with the `COMMAND` text we use in other refs I think it should be upper case here too:

```suggestion
 TEMPLATE
```